index.d.ts

```diff
@@ -33,8 +33,6 @@ export interface Action {
 export interface Column {
   cellStyle?: React.CSSProperties | ((data: any) => React.CSSProperties);
   currencySetting?:{ locale?: string,currencyCode?: string,minimumFractionDigits?:number,maximumFractionDigits?:number};
-  customFilterAndSearch?: (filter: any, rowData: any, columnDef: Column) => boolean;
```

Comment:
You should not remove these lines. Please pull from material-table to your own fork.

Reply:
Sorry , failed to do that, fixed it